client/src/Data/Api.elm

```diff
@@ -2,7 +2,7 @@ module Data.Api exposing
     ( Endpoint(..), get, post, put, patch, delete
     , withJsonBody, withStringResponse, withJsonResponse, withStringErrorHandler, withErrorHandler
```

Comment:
I'm still getting the warning (I think we decide to hide it 🤔), is it intentional ?

(Mainly with address that start with: `123 US HIGHWAY 41 N`)

[Screencast from 08-29-2025 07:43:08 AM.webm](https://github.com/user-attachments/assets/9481db8d-03a2-4e53-aa2e-1aaea2579d12)

Reply:
Updated the `Address` modal to avoid showing these warnings